site/app/Resources/views/user/notification_inbox.html.twig

```diff
@@ -0,0 +1,54 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}
+    Notification Inbox
```

Comment:
Please indent this file by two spaces and double-check all files this time please.

Reply:
Sure!